### PR TITLE
Add circleci builds

### PR DESCRIPTION
**Pull request by mark-ship-it**
Fixes # 
Use (new) publish step in CDK Orb
Update projen to contain our contact info and compatibility with orb

Need feedback on:
our email address
issues building with new orb
Should we disable github workflows? (it's built into projen)

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -0,0 +1,50 @@
+version: 2.1
+
+orbs:
+  cdk: helix/cdk@dev:e52ed15 #0.5.0
```
Comment (ahammond) [inline .circleci/config.yml:4]: `@dev:alpha` will use the last built alpha version. It's kinda like `latest`. :)

File: LICENSE [hunk 2]
```diff
@@ -186,7 +186,7 @@
       same "printed page" as the copyright notice for easier
       identification within third-party archives.
 
-   Copyright 2021 Elad Ben-Israel
+   Copyright 2021 Helix Devops
```
Comment (ahammond) [inline LICENSE:189]:
```suggestion
   Copyright 2021 Elad Ben-Israel
   Copyright 2021 Helix OpCo, LLC
```

**Review by ahammond — APPROVED** [review]

File: .circleci/config.yml (hunk 1 above)
Comment (corrjo) [reply .circleci/config.yml:4]: is there a `latest` tag? 🤩  would love to have that everywhere.

File: .circleci/config.yml [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+          name: "Publish canary build of cdk-watchful"
+          npm_params: --tag canary
+          requires:
+            - Test
+            - Lint
+            - Build
+          context: npm-publish
+          filters:
+            branches:
+              ignore: main
```
Comment (corrjo) [inline .circleci/config.yml:50]: /n

**Review by corrjo — APPROVED** [review]

**Pull request closed without merging**